test/index.ts

```diff
@@ -351,11 +353,10 @@ test(
 			Write: {
 				created_at: Types['Date Time']['Write'];
 				modified_at: Types['Date Time']['Write'];
-				id: Types['Serial']['Write'];
 				a_date: Types['Date']['Write'];
 				a_file: Types['WebResource']['Write'];
-				parent: Parent['Write']['id'];
-				references__other: Other['Write']['id'];
+				parent: Types['Serial']['Write'];
+				references__other: Types['Big Serial']['Write'];
```

Comment:
Can you leave some of these as references please so that we test both references (when not computed) as well as inlined (when computed)